misc/python/materialize/cloudtest/k8s/role_binding.py

```diff
@@ -0,0 +1,28 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from kubernetes.client import V1ObjectMeta, V1RoleBinding, V1RoleRef, V1Subject
+
+from materialize.cloudtest.k8s import K8sRoleBinding
+
+
+class AdminRoleBinding(K8sRoleBinding):
+    def __init__(self) -> None:
+        metadata = V1ObjectMeta(name="admin-binding")
+        role_ref = V1RoleRef(
+            api_group="rbac.authorization.k8s.io", kind="ClusterRole", name="admin"
+        )
+        subjects = [V1Subject(kind="ServiceAccount", name="default")]
+        self.role_binding = V1RoleBinding(
```

Comment:
@benesch What is this for? I lifted it verbatim from the bin/cluster-dev file without knowing what it does in practice.

Reply:
It makes it so `environmentd` can manipulate any Kubernetes objects, like creating storaged and computed statefulsets. By default pods are not allowed to do whatever they want with the broader cluster; this flips that default. In production things are more locked down, but I think it's okay to ignore that for now.

Reply:
Thanks, ack.